radius/traderx.bicep

```diff
@@ -0,0 +1,256 @@
+      image: 'ghcr.io/willtsai/traderx/people-service:latest'
+      ports: {
+        web: {
+          containerPort: 18089
+        }
+      }
+    }
+  }
+}
+
+resource accountservice 'Applications.Core/containers@2023-10-01-preview' = {
+  name: 'account-service'
+  properties: {
+    application: application
+    container: {
+      image: 'ghcr.io/willtsai/traderx/account-service:latest'
+      ports: {
+        web: {
+          containerPort: 18088
+        }
+      }
+      env: {
+        DATABASE_TCP_HOST: database.name
+        PEOPLE_SERVICE_HOST: peopleservice.name
+      }
+    }
+    connections: {
+      peopleservice: {
+        source: peopleservice.id
+      }
+      db: {
+        source: database.id
+      }
+    }
+  }
+}
+
+resource positionservice 'Applications.Core/containers@2023-10-01-preview' = {
+  name: 'position-service'
+  properties: {
+    application: application
+    container: {
+      image: 'ghcr.io/willtsai/traderx/position-service:latest'
+      ports: {
+        web: {
+          containerPort: 18090
+        }
+      }
+      env: {
+        DATABASE_TCP_HOST: database.name
+      }
+    }
+    connections: {
+      db: {
+        source: database.id
+      }
+    }
+  }
+}
+
+resource tradeservice 'Applications.Core/containers@2023-10-01-preview' = {
+  name: 'trade-service'
+  properties: {
+    application: application
+    container: {
+      image: 'ghcr.io/willtsai/traderx/trade-service:latest'
+      ports: {
+        web: {
+          containerPort: 18092
+        }
+      }
+      env: {
+        DATABASE_TCP_HOST: database.name
+        PEOPLE_SERVICE_HOST: peopleservice.name
+        ACCOUNT_SERVICE_HOST: accountservice.name
+        REFERENCE_DATA_HOST: referencedata.name
+        TRADE_FEED_HOST: tradefeed.name
+      }
+    }
+    connections: {
+      db: {
+        source: database.id
+      }
+      peopleservice: {
+        source: peopleservice.id
+      }
+      accountservice: {
+        source: accountservice.id
+      }
+      referencedata: {
+        source: referencedata.id
+      }
+      tradefeed: {
+        source: tradefeed.id
+      }
+    }
+  }
+}
+
+resource tradeprocessor 'Applications.Core/containers@2023-10-01-preview' = {
+  name: 'trade-processor'
+  properties: {
+    application: application
+    container: {
+      image: 'ghcr.io/willtsai/traderx/trade-processor:latest'
+      ports: {
+        web: {
+          containerPort: 18091
+        }
+      }
+      env: {
+        DATABASE_TCP_HOST: database.name
+        TRADE_FEED_HOST: tradefeed.name
+      }
+    }
+    connections: {
+      db: {
+        source: database.id
+      }
+      tradefeed: {
+        source: tradefeed.id
+      }
+    }
+  }
+}
+
+resource webfrontend 'Applications.Core/containers@2023-10-01-preview' = {
+  name: 'web-front-end-angular'
+  properties: {
+    application: application
+    container: {
+      image: 'ghcr.io/willtsai/traderx/web-front-end-angular:latest'
```

Comment:
Is there a way to make this tied to a public or ephemeral container registry namespace?  This seems to be under your account, which seems too specific for the main branch of this project

Reply:
Hi @DovOps - thanks for the suggestion, do you know if it's possible to publish these images to the [TraderX GHCR](https://github.com/orgs/finos/packages?repo_name=traderX) namespace? If so, then I can update these to point to there.

Reply:
@karlmoll  - do you know the answer to this question?

Reply:
@DovOps @willtsai - apologies for missing this, I will confer with @maoo @TheJuanAndOnly99

Reply:
@willtsai - of course we can! Assuming the code is built from the TraderX codebase (or one of its branches) and ll code is released under Apache 2.0) we don't have any compliance issue. It would be awesome to do a `docker scan` (or similar) step prior to the push, to make sure we're not shipping code with known vulnerabilities.

I'd suggest contributing a GitHub Action, so that a maintainer can trigger the build (see [workflow_dispatch trigger](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch)) and we don't need to deal with permissions.

Please let me know if I missed anything or you have further questions/doubts.

Reply:
@maoo -- awesome, i'll look into adding a github action and reach out if i need help triggering it for testing, etc.

Reply:
created another PR for the GitHub Action to add CI for this repo - https://github.com/finos/traderX/pull/222